shared error

### Environment

------------------------------
- Operating System: `Linux`
- Node Version:     `v18.15.0`
- Nuxt Version:     `3.3.2`
- Nitro Version:    `2.3.2`
- Package Manager:  `yarn@1.22.19`
- Builder:          `vite`
- User Config:      `ssr`, `telemetry`, `runtimeConfig`, `app`, `typescript`, `modules`, `lodash`, `components`, `imports`, `alias`, `build`, `css`, `vite`
- Runtime Modules:  `@pinia/nuxt@0.4.7`, `@pinia-orm/nuxt@1.6.5`, `@vueuse/nuxt@9.13.0`, `nuxt-lodash@2.4.1`
- Build Modules:    `-`
------------------------------


### Reproduction

the error appeared after the update, and continues to appear since version 1.6.1
in version 1.5.1 there is no such error

### Describe the bug

appears when the application is launched in dev mode, I did not even try to build

### Additional context

_No response_

### Logs

```shell
ERROR  Error while requiring module @pinia-orm/nuxt: Error: Cannot find module './shared/pinia-orm.34b622d8.cjs'
 Require stack:
 - /app/node_modules/pinia-orm/dist/index.js
  
  ERROR  Cannot start nuxt:  Cannot find module './shared/pinia-orm.34b622d8.cjs'
 Require stack:
 - /app/node_modules/pinia-orm/dist/index.js
 
   Require stack:
   - node_modules/pinia-orm/dist/index.js
   at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
   at Function.resolve (node:internal/modules/cjs/helpers:116:19)
   at _resolve (node_modules/jiti/dist/jiti.js:1:241025)
   at jiti (node_modules/jiti/dist/jiti.js:1:243309)
   at node_modules/pinia-orm/dist/index.js:3:15
   at jiti (node_modules/jiti/dist/jiti.js:1:245784)
   at node_modules/@pinia-orm/nuxt/dist/module.cjs:4:18
   at jiti (node_modules/jiti/dist/jiti.js:1:245784)
   at requireModule (node_modules/@nuxt/kit/dist/index.mjs:286:26)
   at normalizeModule (node_modules/@nuxt/kit/dist/index.mjs:472:77)
```
```

--- a/packages/nuxt/build.config.ts
+++ b/packages/nuxt/build.config.ts
@@ -4,6 +4,5 @@ import { defineBuildConfig } from 'unbuild'
 export default defineBuildConfig({
   rollup: {
     emitCJS: true
-  },
-  externals: ['pinia', '@nuxt/kit-edge', '@nuxt/types', 'pinia-orm', '@pinia-orm/normalizr']
+  }
 })

--- a/packages/nuxt/package.json
+++ b/packages/nuxt/package.json
@@ -24,7 +24,7 @@
     }
   },
   "main": "./dist/module.cjs",
-  "types": "./dist/types.d.ts",
+  "types": "./dist/module.d.ts",
   "files": [
     "dist",
     "LICENSE",
@@ -54,7 +54,7 @@
     "@pinia/nuxt": "^0.4.7",
     "@types/prettier": "^2.7.2",
     "eslint": "^8.36.0",
-    "nuxt": "^3.3.1",
+    "nuxt": "^3.3.2",
     "pinia": "^2.0.33",
     "pinia-orm": "^1.0.0",
     "prettier": "^2.8.7",

--- a/playgrounds/nuxt3/package.json
+++ b/playgrounds/nuxt3/package.json
@@ -12,7 +12,7 @@
     "@pinia/nuxt": "^0.4.7",
     "nuxt": "^3.3.1",
     "pinia": "^2.0.20",
-    "pinia-orm": "^1.5.1"
+    "pinia-orm": "^1.6.5"
   },
   "dependencies": {
     "nanoid": "^4.0.0",

